packages/in_app_purchase/in_app_purchase/pubspec.yaml

```diff
@@ -15,6 +15,8 @@ flutter:
         default_package: in_app_purchase_android
       ios:
         default_package: in_app_purchase_storekit
+      macos:
+        default_package: in_app_purchase_storekit
```

Comment:
The minimum version of `in_app_purchase_storekit` below should be updated to 0.3.4, so that macOS support is guaranteed to be present.